crates/proof-of-sql/src/sql/ast/projection_expr_test.rs

```diff
@@ -0,0 +1,357 @@
+use crate::{
+    base::{
+        database::{
+            owned_table_utility::*, ColumnField, ColumnRef, ColumnType, OwnedTable,
+            OwnedTableTestAccessor, RecordBatchTestAccessor, TableRef, TestAccessor,
```

Comment:
NIT: Let's used `OwnedTableTestAccessor` instead of `RecordBatchTestAccessor` and avoid any arrow within these tests.